src/lib/convert.ts

```diff
@@ -48,6 +46,7 @@ export const convert = async (convertionOptions: ConvertionOptions): Promise<voi
         );
       }
     }
+    types = types.substring(0, types.length - 3) + ';';
```

Comment:
Why, types.length - 3?

Reply:
I remove the last 3 chars eg: ' | '.
Maybe it's not easy to reason about ...

Maybe we can create a var "typeDelimitor" equal to ' | ' and replace 3 by typeDelimitor.length.

Reply:
Yes, that's a good idea - it would probably be more readable.

Reply:
Done